docs/docs/References/Plugin_Operations.md

```diff
@@ -593,6 +594,68 @@ def linked_mount_specification(staged_source, repository):
   return MountSpecification([mount], ownership_spec)
 ```
 
+## Virtual Source Initialize
+
+Initializes a brand-new [empty VDB](Glossary.md#empty-vdb). As with all VDBs, this new dataset will have access to mounted Delphix Engine storage, but of course there will be no data there at first.
```

Comment:
again not sure if "of course" part really make sense here. I think it's reasonable just to have the sentence say something like:
```
Initializes a brand-new empty VDB. As with all VDBs, this new dataset will have access to mounted Delphix Engine storage. The VDB will however start out with no data.```

or something along these lines?